recipes/jsonformoderncpp/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class JsonformoderncppConan(ConanFile):
+    name = "jsonformoderncpp"
+    homepage = "https://github.com/nlohmann/json"
+    description = "JSON for Modern C++ parser and generator."
+    topics = ("conan", "jsonformoderncpp", "json", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    license = "MIT"
+
+    options = {
+        "multiple_headers": [True, False],
```

Comment:
You are missing a default value for the option. [We encourage to test the recipes locally first](https://github.com/conan-io/conan-center-index/wiki#test-the-recipe-locally).

What is the `JSON_MultipleHeaders` about?

Reply:
The local test was successful at the time tested.
I will resolve later these days.

Reply:
Multiple headers are defining if one header file or multiple are used afaik.

Reply:
Sorry, seems like my comment was confusing when combined with how GitHub is displaying inline comments. 

I meant to only remove the `exports = ["LICENSE.md"]` line :)